### ISSUE TITLE
"karma": ">=0.9" - is it correct?

### ISSUE THREAD

**Issue by dragonflypl** [karma-runner/karma-phantomjs-launcher#107]
Hi, 

it this peerDependency correct ? I mean, on https://github.com/karma-runner/karma/blob/master/package.json the version is 0.13.22. So propably I don't understand something. Anyway, how can I get rid of peerdepenedency warning go away when installing karma-phantomjs-launcher?

**Comment by dignifiedquire** [comment]
Yes this is correct, we support all karma versions starting from `0.9` upwards. You will need to actually install `karma` for the warning to go away.
